docs/workflows/best-practices-osh.md

```diff
@@ -0,0 +1,398 @@
+  - A licensing risk will usually only appear when a library has been newly introduced.
+  Likely actions:
+    - [Updating a library](#updating-a-library) 
+
+
+### 5. Handling detected vulnerabilities
+Security risks of a certain framework or library should be determined based on at least the following aspects: 
+​- The severity level of known vulnerabilities for the artifact​
+- The mission of the components where the framework or the library will be applied​: Non mission critical / ​Mission ​critical
+- The outside visibility: ​External Facing or Distributed​ 
+  > [? is this the right classification?]
+  - public facing systems need to be tackled first, and should not have any vulnerabilities of risk category _high_ and higher.
+
+When a vulnerability is found, it will be remediated within a specified time period.
+  The table below is a suggestion how to specify this, depending on the criticality of the system:
+  | CVSSv3 Range | Label | Remediation Deadline Critical | Remediation Deadline Regular |
+  | --- | --- | --- | --- |
+  | 9.0 – 10.0 | Critical | Within 1 working day | Within 1 working day | 
+  | 7.0 – 8.9 | High | Within 14 days | Within 30 days |
+  | 4.0 – 6.9 | Medium | Within 60 days | Within 90 days |
+  | 0.1 – 3.9 | Low | Within 1 year | Within 1 year |
+  > TODO: check that the above table makes sense, especially the added last column
+
+- If no remediation is available, do a risk assessment which will have one of 3 outcomes:
+  - If we find that the vulnerability does not pose any actual risk, we will ‘allowlist’ it. This requires CISO approval. This allowlist will be reviewed as part of the half-yearly measurement cycle.
+  - We will mitigate the risk in some other way. If, for example, we do not want to allowlist. the entire library because of its importance but the vulnerability is limited to a single method, we can test for the use of that method and fail the pipeline in that case.
+  - [In](http://3.in/) extreme cases, we will shut down the application until the vulnerability is resolved.
+  - note: when addressing the vulnerabilities of a specific library by updating to a newer version, that always also improves the freshness
+  
+
+### 6. Handling detected license issues [ROUGH DRAFT]
+- Libraries must have an acceptable license.
+  - This may be a paid license or an acceptable open-source license. [Company] maintains a list of common licenses used in free and open-source software (FOSS); if a library has a license listed as acceptable, it can be used. Otherwise, see if an alternative is available, or contact the [applicable role] to discuss whether the license is acceptable.
+- See also the License Risk Evaluation Framework slide at the bottom of the wiki page: [OSH interpret results wiki page: https://softwareimprovementgroup.atlassian.net/wiki/spaces/SSM/pages/50317951024/How+to+interpret+Open-Source+health+findings](https://workflowy.com/#/c9c4745b6b6b)
```

Comment:
In our documentation, we cannot refer to the wiki (at least, we need to make it clear that 'regular' users can't use the link.)

Reply:
indeed, this was a 'note to self'